[DOCS] Endpoint artifact docs clarification (#2516)
elastic/security-docs: docs/management/admin/event-filters.asciidoc

> Event filters allow you to filter endpoint events that you do not need or want stored in {es} -- for example, high-volume events. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.

Reviewer: This isn't true and it feels like a non sequitur.  What is this sentence trying to convey?

Author: @ferullo I agree, that does seem out of place. I think it's related to a Tip a little further down the page, where we suggest filtering events to only display Endpoint events ("In the KQL search bar, enter the following query: `event.dataset : endpoint.events.network`.").

If that's not an accurate way to filter endpoint events, we should probably revise this TIp too. Is there a better query or method we could suggest?

Reviewer: Ah, ok, that's an accurate tip on how to filter for Endpoint network events. All Endpoint _network_ events have that field/value.

Author: Thanks for clarifying, @ferullo. I just [pushed a change](https://github.com/elastic/security-docs/pull/2516/commits/d64b7fc997315911b3fafde715801ffad46a3d74) to remove the non sequitur from the first paragraph and revise the Tip below.

> Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors event data to detect and prevent possible threats, but the event data just isn't being written to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.

Reviewer 2: ```suggestion
Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors filtered events to detect and prevent possible threats, but prevents them from being written to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```

Reviewer 2: Does it write them to ES if they end up triggering an alert? 🤔

Author: I assume only the alerts would be written to ES. 

A modified version of your revision (and my modification here is actually based on your revision to the same phrasing in the overall artifacts table above).
```suggestion
Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors events to detect and prevent possible threats, but without writing event data to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```

NOTE: You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].

> IMPORTANT: Since an event filter blocks an event from streaming to {es}, be conscious of event filter conditions you set and any existing rule conditions. If there is too much overlap, the rule may run less frequently than specified and, therefore, will not trigger the corresponding alert for that rule. This is the expected behavior of event filters.

Reviewer 2: ```suggestion
IMPORTANT: Since an event filter blocks events from streaming to {es}, be conscious of event filter conditions you set and any existing rule conditions. If there is too much overlap, the rule may run less frequently than specified and therefore not trigger the corresponding alert for that rule. This is the expected behavior of event filters.
```
